tdrs-frontend/src/utils/canViewAdmin.js

```diff
@@ -0,0 +1,11 @@
+const isMemberOfOne = (user, groupNames) =>
+  user?.roles?.some((role) => groupNames.includes(role.name))
+
+const userAccessRequestApproved = (user) =>
+  user?.['access_request'] && user?.roles?.length > 0
+
+const canViewAdmin = (user) =>
+  userAccessRequestApproved(user) &&
+  isMemberOfOne(user, ['Developer', 'OFA System Admin', 'ACF OCIO'])
```

Comment:
@n0remac `OFA Admin` also has access to backend, but the admin tab hasnt been visible for this group for some reason. a user with this role can submit data on behalf of any STT.  if its easy to include the tab and group in this, that'd be great. else, we can spin up a new ticket.